### PR TITLE
Fix egg import - Implode expects an array

### LINKED ISSUE #3034: Egg Import ends in an Error 500, because of file_denylist in Eggfile
**Background (please complete the following information):**
* Panel or Daemon:  
* Version of Panel/Daemon: 1.2.2 / 1.2.3 
* Server's OS: Debian 10 
* Your Computer's OS & Browser: Windows 10 / Chrome 88 

**Describe the bug**

When u create an Egg with Panel 1.2.2 and Wings 1.2.3, there will be a new Variable added -> `"file_denylist": "",`

After Export and try to import the Egg into the same Panel, it ends in an Error 500.

When you delete the Line:

`"file_denylist": "",`

you can import the Egg without problems

**To Reproduce**
1. Create an Egg with Panel 1.2.2 and Wings 1.2.3
2. Export it
3. re-import the egg

### PR DESCRIPTION
**Pull request by revij**
Fixes #3034

### CODE REVIEW

File: app/Services/Eggs/Sharing/EggImporterService.php [hunk 1]
```diff
@@ -85,7 +85,7 @@ public function handle(UploadedFile $file, int $nest): Egg
             // Maintain backwards compatability for eggs that are still using the old single image
             // string format. New eggs can provide an array of Docker images that can be used.
             'docker_images' => object_get($parsed, 'images') ?? [object_get($parsed, 'image')],
-            'file_denylist' => implode(PHP_EOL, object_get($parsed, 'file_denylist') ?? []),
+            'file_denylist' => implode(PHP_EOL, array(object_get($parsed, 'file_denylist') ?? [])),
```
Comment (ArnaudLier) [inline app/Services/Eggs/Sharing/EggImporterService.php]: Use [] instead of array()?

**Review by ArnaudLier — CHANGES_REQUESTED** [review]

**Comment by revij** [comment]
Hmm never mind, just realised that it only fixes the import if file_denylist is not an array.

**Comment by gOOvER** [comment]
Great, ty. Works for me :)

**Comment by DaneEveritt** [comment]
Closed in https://github.com/pterodactyl/panel/commit/b00def253746d1c96ba2c4bdc232d436fd7b7fee with a proper fix.

**Pull request closed without merging**